plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/templates/appengine/AppEngineTemplateUtilityTest.java

```diff
@@ -79,7 +79,7 @@ public void testCreateFileContent_helloAppEngine()
       throws CoreException, IOException {
     String fileLocation = testFile.getLocation().toString();
     Map<String, String> dataMap = new HashMap<String, String>();
-    dataMap.put("package", "package com.example");
+    dataMap.put("package", "com.example");
```

Comment:
The previous code (`package X`) was necessary to handle when the user selects the default package.  It's unfortunately a valid package to select and we need to support it.  Can freemarker support optional template pieces, like in sh:
```
${package:+package ${package};}
```
?